packages/devtools_app/lib/src/screens/inspector_v2/inspector_controller.dart

```diff
@@ -645,32 +586,15 @@ class InspectorController extends DisposableController
       isSummaryTree: isSummaryTree,
```

Comment:
we should be able to remove all variables / references to the concept "is summary tree" now right?

Reply:
Good point! Removed all references except where it is being used except for in inspector service methods (since that is shared with the legacy inspector)